webapp/src/components/EndpointsTable/index.js

```diff
@@ -9,20 +10,53 @@ import TableHead from '@mui/material/TableHead'
 import TableRow from '@mui/material/TableRow'
 import Typography from '@mui/material/Typography'
 
+import HealthCheck from '../HealthCheck'
+
+import styles from './styles'
+import EndpointInfo from './EndpointInfo'
+
+const useStyles = makeStyles(styles)
+
 const EndpointsTable = ({ producers }) => {
+  const classes = useStyles()
   const { t } = useTranslation('endpointsListRoute')
 
-  const CellList = ({ producer, endpointType }) => {
+  const getStatus = (endpoint) => {
+    if (endpoint.response.status === undefined) return
 
+    if ((new Date() - new Date(endpoint.head_block_time)) / 60000 <= 3) {
```

Comment:
may be a good idea to set the time difference as a aparameter?

Reply:
I think this option is better, thanks